docs/source/technical_design/hardening.rst

```diff
@@ -9,9 +9,9 @@ DJANGO_ALLOWED_HOSTS
 
 Django uses the ``DJANGO_ALLOWED_HOSTS`` setting to determine which host/domain names it can serve. This is a security measure to prevent HTTP Host header attacks, which are possible even under many seemingly-safe web server configurations.
 
-The default value for this setting is ``*``, which allows all hosts. You should always set this to the domain and subdomain names that your application uses. For example, if your application is available at ``example.com`` and ``subdomain.example.com``, you should set ``DJANGO_ALLOWED_HOSTS`` to ``example.com subdomain.example.com`` (space separated).
+The default value for this setting is ``*``, which allows all hosts. You should always set this to the domain and subdomain names that your application uses. For example, if your application is available at ``example.com`` and ``subdomain.example.com``, you should set ``DJANGO_ALLOWED_HOSTS`` to ``example.com subdomain.example.com`` (comma separated).
```

Comment:
Maybe change ``example.com subdomain.example.com`` into ``example.com,subdomain.example.com`` to reflect the changes?

Reply:
Great find! Updated